src/plugins/subsystems/drm/drm.sh

```diff
@@ -215,10 +223,18 @@ function gui_control()
     isolate_target='graphical.target'
     vt_console=1
     gui_control_cmd="${configurations[gui_on]}"
-  else
+  elif [[ "$operation" == 'OFF' ]]; then
     isolate_target='multi-user.target'
     vt_console=0
     gui_control_cmd="${configurations[gui_off]}"
+  elif [[ "$operation" == 'ON_AFTER_REBOOT']]; then
+    isolate_target='graphical.target'
+    vt_console=1
```

Comment:
why do we need this?

Reply:
We add another setting like ON and OFF to [ON|OFF]_AFTER_REBOOT. So, this is to add more verifications to select the right command.

Reply:
I meant `vt_console=1`, do we really need it?

Reply:
I only followed the pattern from the other if.